Build broken without OMP

**Describe the bug**

Currently, setting -DOPENMP=OFF does not fully remove OpenMP and vcflib is only buildable with OMP. It seems like when vcfwave.cpp was added, it unconditionally calls OpenMP functionality.

**To Reproduce**

Try to build on a system without OMP:

```
git clone --recursive https://github.com/vcflib/vcflib.git
cd vcflib
mkdir -p build && cd build
cmake -DCMAKE_BUILD_TYPE=Debug -DZIG=OFF -DOPENMP=OFF ..
cmake --build .
cmake --install .
```

Fail:
```
Undefined symbols for architecture arm64:
  "_omp_set_num_threads", referenced from:
      _main in vcfwave.cpp.o
ld: symbol(s) not found for architecture arm64
```

**Expected behavior**

When building vcflib with OPENMP=OFF, it should build without requiring OpenMP.

--- a/src/vcfwave.cpp
+++ b/src/vcfwave.cpp
@@ -14,7 +14,9 @@
 #include <vector>
 
 #include <getopt.h>
+#ifdef WFA_PARALLEL
 #include <omp.h>
+#endif
 
 #include "Variant.h"
 #include "vcf-wfa.h"
@@ -181,7 +183,9 @@ int main(int argc, char** argv) {
         }
     }
 
+    #ifdef WFA_PARALLEL
     omp_set_num_threads(thread_count);
+    #endif
 
     off_t file_size = -1;
 

